lib/grandpa/vote_message.go

```diff
@@ -230,6 +217,7 @@ func (s *Service) validateVoteMessage(from peer.ID, m *VoteMessage) (*Vote, erro
 		s.prevotes.Store(pk.AsBytes(), just)
 	case precommit:
 		s.precommits.Store(pk.AsBytes(), just)
+		logger.Debugf("stored precommit: num=%d", s.lenVotes(precommit))
```

Comment:
nit why not more human string like

```suggestion
		logger.Debugf("stored %d precommit votes", s.lenVotes(precommit))
```